app/src/test/java/org/cru/godtools/databinding/ToolDetailsFragmentBindingTest.kt

```diff
@@ -32,6 +34,7 @@ class ToolDetailsFragmentBindingTest {
         binding.manifest = MutableLiveData<Manifest>(manifest)
         binding.executePendingBindings()
 
+        assumeTrue(BuildConfig.DEBUG)
```

Comment:
generally I put the `assume` calls at the beginning of the test because it prevents the entire test from running when the `assume` isn't satisfied